common/src/main/scala/hmda/publication/KafkaUtils.scala

```diff
@@ -74,4 +97,33 @@ object KafkaUtils {
 
   }
 
+  def consumeInstitutionRecords(topic: String,
+                                f: Future[Done],
+                                parallelism: Int)(
+      implicit system: ActorSystem,
+      materializer: ActorMaterializer,
+      ec: ExecutionContext) = {
+
+    val config = system.settings.config.getConfig("akka.kafka.consumer")
+
+    val consumerSettings: ConsumerSettings[String, InstitutionKafkaEvent] =
+      ConsumerSettings(config,
+                       new StringDeserializer,
+                       new InstitutionKafkaEventsDeserializer)
+        .withBootstrapServers(kafkaHosts)
+        .withGroupId(UUID.randomUUID().toString)
+        .withProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest")
+
+    Consumer
+      .committableSource(consumerSettings, Subscriptions.topics(topic))
```

Comment:
Probably worth adding a second, "admin" topic so that this event can be triggered on demand without sending additional, repeated events to the journal

Reply:
I was messing around with this, but since each message holds a serialized Institution, I don't think you can send a custom message using the command line (unlike strings).  Is there a way to pass in some `Institution(Bank0,test,test)` using the console producer?